app/scripts/controllers/user-storage/encryption.ts

```diff
@@ -28,9 +41,13 @@ class EncryptorDecryptor {
 
   #ALGORITHM_KEY_SIZE: number = 16; // 16 bytes
 
-  #PBKDF2_SALT_SIZE: number = 16; // 16 bytes
+  #SCRYPT_SALT_SIZE: number = 16; // 16 bytes
 
-  #PBKDF2_ITERATIONS: number = 900_000;
+  #SCRYPT_N: number = 2 ** 17; // CPU/memory cost parameter (must be a power of 2, > 1)
+
+  #SCRYPT_r: number = 8; // Block size parameter
+
+  #SCRYPT_p: number = 1; // Parallelization parameter
```

Comment:
I believe N and P are the work factor levers.

Using OWASP configurations defined here.
https://cheatsheetseries.owasp.org/cheatsheets/Password_Storage_Cheat_Sheet.html#scrypt

This seems "okay" from some automated testing, will need to actually hook it up to see the UX impact.